flow360/component/flow360_params/solvers.py

```diff
@@ -556,6 +557,7 @@ def update_model(self):
             "limitPressureDensity": self.limit_pressure_density,
             "numericalDissipationFactor": self.numerical_dissipation_factor,
             "lowMachPreconditioner": self.low_mach_preconditioner,
+            "lowMachPreconditionerThreshold": self.low_mach_preconditioner_threshold,
```

Comment:
Why the lowMachPreconditioner is in the legacy model? This is a new feature right?